src/lib/stringIndices.js

```diff
@@ -1,16 +1,19 @@
-  let stringIndices = (userString, userNumber) => {
+
+let stringIndices = (userString, userNumber) => {
     let newArr = []
     let separate = ""
+    let emptyString = " "
+
+    if(userNumber > newArr || userNumber < 0){
+      return emptyString
+    }else{
+      if(userString.indexOf(" ") !== -1){
+        let separate = userString.split(' ')
+        let newArr = Array.from(separate)
```

Comment:
This should not be necessary - the `.split` function should be returning an array already?